VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsBaseFragment.java

```diff
@@ -68,10 +59,26 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                     thread.setRead(true);
                     discussionPostsAdapter.getView(position, view, parent);
                 }
-
-                CourseDiscussionPostsBaseFragment.this.onItemClick(thread, parent, view,
-                        position, id);
             }
         });
     }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+
+        if (isRestart) {
+        /*
+        If the activity/fragment needs to be reinstantiated upon restoration,
+        then in some cases the onRestart() callback maybe invoked before view
+        initialization, and thus the controller might not be initialized, and
+        therefore we need to guard this with a null check.
+         */
```

Comment:
This should probably be indented to be at the same level as the code in the `if` block. Also there's an extra space before the `*/`. Java coding conventions also recommend prefixing each line in comment blocks with asterisks, which makes it easier to identify it at a glance as a comment. For example:

```` java
/*
 * Here is a block comment.
 */
````